### ISSUE TITLE
Error and Signs not working anymore

### ISSUE THREAD

**Issue by RemcoPl** [WirelessRedstoneGroup/WirelessRedstone#213]
## Bug report

**I'm using version:** 4.0.1
**I'm on Minecraft:** 1.13
**I'm running:** Spigot

**Seen results:**  
Get a warning and sign doen't work anymore

**Expected results:**  
Tell use what should have happened

**How to reproduce:**  
i have been update form v3 to v4 on 1.13 and then this happends

**Additional info:**  
Console error https://pastebin.com/1bYKwwiw

**Comment by RemcoPl** [comment]
Btw this error is spamming in my console

**Comment by Bart-0110** [comment]
Hello, there are some unknowns to the scenario. Can you answer the next questions:

* Can you restart your server? 
* Does this still happen then? 
* Did you manually edit the SQLite file or did the plugin make all the changes?

**Comment by RemcoPl** [comment]
Can you restart your server? I have done that more then 1 times
Does this still happen then? Yes
Did you manually edit the SQLite file or did the plugin make all the changes? No i dont edit the SQLite manual

**Comment by Bart-0110** [comment]
Can you send your SQLite file? I'm having trouble finding the problem.

**Comment by RemcoPl** [comment]
[WirelessRedstoneDatabase.zip](https://github.com/WirelessRedstoneGroup/WirelessRedstone/files/3512047/WirelessRedstoneDatabase.zip)

Good luck

**Comment by RemcoPl** [comment]
And yes it is a .zip because github doen't support that kind of file

**Comment by RemcoPl** [comment]
did you found anything yet?

**Comment by Bart-0110** [comment]
Can you try this new beta? https://github.com/WirelessRedstoneGroup/WirelessRedstone/releases/tag/v4.0.2-beta.4

**Comment by RemcoPl** [comment]
That help the error out

But we have now a problem.
that old signs we have deleted.
after a restart, there are back in place.

**Comment by Bart-0110** [comment]
What do you mean exactly? You removed a sign by breaking the sign?

**Comment by RemcoPl** [comment]
yes we break the sign and after \restart or anything thats reloading the plugin.
the signs are spawning back and are different what we have put there


sorry about the delay

**Comment by RemcoPl** [comment]
.......
Anything found on the new bugg?

**Comment by Bart-0110** [comment]
Can you give me the channel name of the sign that doesn't get deleted? I think that there is a name error.

**Comment by RemcoPl** [comment]
dive_vloer
inrang_dive
power_nemo

**Comment by RemcoPl** [comment]
i have been fixed the issue
but i have found this
![2019-09-02_21 47 06](https://user-images.githubusercontent.com/31757667/64131802-618a0080-cdcb-11e9-8a32-d5b2ea9ca4df.png)

**Comment by Bart-0110** [comment]
Can you use v4.0.4? https://github.com/WirelessRedstoneGroup/WirelessRedstone/releases/tag/v4.0.4
